modules/virt-importing-vm-wizard.adoc

```diff
@@ -101,7 +102,20 @@ ifdef::virt-importing-vmware-vm[]
 ** *Source*: For example, *Import Disk*.
 ** *Size*
 ** *Interface*
-** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class to create the virtual machine.
+** *Storage Class*: If you do not select a storage class, the default storage class is used for the VM disk and the v2v-conversion-template disk. The v2v-conversion-template disk is a 2 GB disk that is used as scratch space for copying the VM disk.
++
+The following storage classes are supported for VMware VM import:
+
+*** VM disk: *ocs-storagecluster-ceph-rbd (ceph-rbd)*
+*** v2v-conversion-template disk: *ocs-storagecluster-ceph-rbd (ceph-rbd)*
++
+_or_
+
+*** VM disk: *NFS*
+*** v2v-conversion-template disk: *hostpath-provisioner (HPP)*
++
+Other storage classes may work although they are not officially supported.
```

Comment:
I think this sentence might encourage customers to try unsupported storage classes, so I would personally remove it. 

If you keep it in, s/may/might (ISG)
s/work although/work, but

Reply:
We don't really care if they try it. If it works, that's great. If it doesn't, the import won't work and they'll have to use a different storage class. They won't break anything by using an unsupported storage class.

Reply:
@djfjeff Hey, can you please confirm that this is okay with you and the support team? Thanks!

Reply:
@ousleyp I am OK with that (including the suggestions you made 2 comments above). I think we should have this mention and not leave this unclear.

Reply:
Thanks @djfjeff! 

@apinnick, I'll be happy to merge this PR after you change this line to "Other storage classes might work, but they are not officially supported."

Reply:
@ousleyp Done